graphql/admin/assign.go

```diff
@@ -39,7 +39,7 @@ func resolveAssign(ctx context.Context, m schema.Mutation) (*resolve.Resolved, b
 	num := &pb.Num{Val: input.Num}
 	switch input.What {
 	case uid:
-		resp, err = worker.AssignUidsOverNetwork(ctx, num)
+		resp, err = worker.AssignIDsOverNetwork(ctx, num)
```

Comment:
You will need to assign Type inside num, and now the whole switch case can be simplified